berkeley-mobile/Drawer/DrawerViewController.swift

```diff
@@ -37,8 +39,16 @@ class DrawerViewController: UIViewController {
         
         setupBackgroundView()
         setupGestures()
+        
+        let nc = NotificationCenter.default
+        nc.addObserver(self, selector: #selector(collapseTopDrawer), name: Notification.Name(TabBarController.homePressedCollapseMessage), object: nil)
     }
     
+    @objc func collapseTopDrawer() {
+        delegate.moveDrawer(to: .collapsed, duration: 0.5)
```

Comment:
small nitpick but should remove duration: 0.5 (just have delegate.moveDrawer(to: .collapsed))
default move duration is 0.5 anyway and we should keep all the default move durations in one place